internal/apis/controlplane/kubeadm/v1alpha3/conversion_test.go

```diff
@@ -82,3 +83,10 @@ func kubeadmClusterConfigurationFuzzer(obj *upstreamv1beta1.ClusterConfiguration
 	// ClusterConfiguration.UseHyperKubeImage has been removed in v1alpha4, so setting it to false in order to avoid v1alpha3 --> v1alpha4 --> v1alpha3 round trip errors.
 	obj.UseHyperKubeImage = false
 }
+
+func kubeadmControlPlaneSpecFuzzer(obj *controlplanev1.KubeadmControlPlaneSpec, c fuzz.Continue) {
+	c.FuzzNoCustom(obj)
```

Comment:
Shouldn't we implement conversion instead?

(applies to all changes in conversion_test.go in other packages as well)